packages/webcomponents/src/components/business-form-stepped/business-form-stepped.tsx

```diff
@@ -21,26 +16,12 @@ export class BusinessFormStepped {
   @Prop() businessId: string;
   @Prop() testMode: boolean = false;
   @Prop() hideErrors?: boolean = false;
-  @State() isLoading: boolean = false;
+  @State() formLoading: boolean = false;
+  @State() serverError: boolean = false;
```

Comment:
We could consider checking if `errorMessage` has a message, then this state wouldn't be necessary.

Reply:
Good call!